src/encoded/static/components/objectutils.js

```diff
@@ -654,21 +628,108 @@ ImageWithFallback.propTypes = {
 
 
 /**
- * Display internal tag badges from search results.
+ * Maps an internal tag to the corresponding image file name. Exported for Jest testing.
```

Comment:
Some say you should never modify code to appease a test. If you subscribe to that belief, should the export be removed and internalTagsMap-const replicated in Jest?

Reply:
That would mean we would need to keep the copy and the main in sync, which seems counter productive.